src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -9322,4 +9219,21 @@ Real64 CalcASTMC1340ConvCoeff(EnergyPlusData &state, int const SurfNum, Real64 c
     return h;
 }
 
+ConvectionConstants::SurfConvOrientation GetSurfConvOrientation(Real64 const &Tilt)
+{
```

Comment:
No need to pass `Tilt` by reference, just extra cost.

Reply:
Thanks for catching that. I'll change it.